_2020/command-line.md

```diff
@@ -371,10 +371,10 @@ This is called _port forwarding_ and it
 comes in two flavors: Local Port Forwarding and Remote Port Forwarding (see the pictures for more details, credit of the pictures from [this StackOverflow post](https://unix.stackexchange.com/questions/115897/whats-ssh-port-forwarding-and-whats-the-difference-between-ssh-local-and-remot)).
 
 **Local Port Forwarding**
-![Local Port Forwarding](https://i.stack.imgur.com/a28N8.png)
+![Local Port Forwarding](https://github.com/missing-semester/missing-semester/raw/master/static/media/images/local-port-forwarding.png)
```

Comment:
We don't want to load images from GitHub raw this way, we want to use a relative URL like we do for other media (just `/media/images/...`).

Reply:
Sure thing! I was not sure if the relative path would work, since I'm not fully familiar with the technical side of the project. Relative path it is!